packages/swap/src/utils/statechain.ts

```diff
@@ -35,22 +41,34 @@ export const getSwapRateV3 = async ({
       }
     : undefined;
 
+  const commonParams: [
+    UncheckedAssetAndChain,
+    UncheckedAssetAndChain,
+    `0x${string}`,
+    number,
+    { number_of_chunks: number; chunk_interval: number } | undefined,
+  ] = [
+    getAssetAndChain(srcAsset),
+    getAssetAndChain(destAsset),
+    hexEncodeNumber(depositAmount),
+    brokerCommissionBps ?? 0,
+    dcaParameters,
+  ];
+
+  const additionalOrders = limitOrders?.filter((order) => order.LimitOrder.sell_amount !== '0x0');
```

Comment:
```suggestion
  const additionalOrders = limitOrders?.filter((order) => BigInt(order.LimitOrder.sell_amount) > 0);
```
not sure if we validate negative amounts before or not

Reply:
it's fine to keep it checking just the strings, this actually shouldn't be an issue anymore but it's a safeguard now